DJHelper/DJHelper/Helpers/HelperExtensions.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  HelperExtensions.swift
+//  DJHelper
+//
+//  Created by Michael Flowers on 6/2/20.
+//  Copyright © 2020 craigswanson. All rights reserved.
+//
+
+import UIKit
+
+extension Date {
+    func stringFromDate() -> String {
+        let formatter = DateFormatter()
+        formatter.dateFormat = "M/d/yyyy h:mm a"
+        return formatter.string(from: self)
+    }
+}
+
+extension String {
+    func dateFromString() -> Date? {
+        let formatter = DateFormatter()
+        formatter.dateFormat = "M/d/yyyy h:mm a"
+        return formatter.date(from: self)
+    }
+}
+
+extension String {
+    func eventDateFromString() -> Date? {
+        let formatter = DateFormatter()
+        formatter.dateFormat = "yyyy-MM-dd'T'HH:mm:ss.SSSZ"
+        return formatter.date(from: self)
+    }
+}
+
+extension UIView {
+    func shake() {
+        let view = self
+        let propertyAnimator = UIViewPropertyAnimator(duration: 0.4, dampingRatio: 0.2) {
+            view.layer.borderColor = UIColor.red.cgColor
+            //move it left by 8 pix
+            view.transform = CGAffineTransform(translationX: -8, y: 0)
+        }
+        propertyAnimator.addAnimations({
+            //return it back to its original position
+            view.transform = CGAffineTransform(translationX: 3, y: 0)
+            view.layer.borderColor = UIColor.green.cgColor
+        }, delayFactor: 0.4)
+        propertyAnimator.startAnimation()
+    }
+}
+
+extension EventController {
```

Comment:
I think having a file to extend EventController is a good idea. That file is already long. We can move this into its own file later.